devel/howdoi/Portfile

```diff
@@ -4,33 +4,30 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                howdoi
-version             1.1.9
+version             2.0.19
 categories          devel python
-platforms           darwin
 license             MIT
 supported_archs     noarch
-
-python.default_version 27
-
 maintainers         nomaintainer
 
 description         instant coding answers via the command line
 long_description    {*}${description}
 homepage            https://pypi.python.org/pypi/howdoi
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 
-distname            ${python.rootname}-${version}
+checksums           rmd160  bb99d6f9638be1ea3e3e51bca244e1c66ef76164 \
+                    sha256  4baa5270a98e9d001094d873a5a1b6a32eac021009cfc40cd9aeede633cb7556 \
+                    size    27109
 
-checksums           rmd160  9c1fdc3dfd03fd717b4978b05928bc0d565f6f10 \
-                    sha256  2b4a0ebc39c9e0489a17ef73e3eec7dd4a2975e07b995406887c639481da2bb5
+python.default_version 39
 
 depends_build-append \
-    port:py${python.version}-setuptools
-
-depends_lib-append \
-    port:py${python.version}-requests \
-    port:py${python.version}-requests-cache \
-    port:py${python.version}-pygments \
-    port:py${python.version}-pyquery
-
-livecheck.type      pypi
+                    port:py${python.version}-setuptools
+
+depends_lib-append  port:py${python.version}-requests \
+                    port:py${python.version}-requests-cache \
+                    port:py${python.version}-pygments \
+                    port:py${python.version}-pyquery \
+                    port:py${python.version}-cssselect \
+                    port:py${python.version}-appdirs \
+                    port:py${python.version}-lxml \
+                    port:py${python.version}-keep
```

Comment:
ah, we typically also sort the dependencies alphabetically

Reply:
Sorted.